### PR TITLE
complete DeltaInt = Int

### PR DESCRIPTION
**Pull request by ecavallo**
@ice1000 was almost there, I just had to remove an `hcomp` face.

### CODE REVIEW

**Comment by ecavallo** [comment]
I think it should be possible to write
```
deltaIntRet : ∀ a → fromInt (toInt a) ≡ a
deltaIntRet (x ⊖ 0) = refl
deltaIntRet (0 ⊖ suc y) = refl
deltaIntRet (suc a ⊖ suc b) = deltaIntRet (a ⊖ b) ∙ cancel a b
deltaIntRet (cancel a b i) j = compPath-filler (deltaIntRet (a ⊖ b)) (cancel a b) i j
```
but the last clause doesn't check for reasons I don't understand. If I break it out into a `where`
```
deltaIntRet : ∀ a → fromInt (toInt a) ≡ a
deltaIntRet (x ⊖ 0) = refl
deltaIntRet (0 ⊖ suc y) = refl
deltaIntRet (suc x ⊖ suc y) = deltaIntRet (x ⊖ y) ∙ cancel x y
deltaIntRet (cancel a b i) = test i
  where
  test : PathP (λ i → fromInt (toInt (a ⊖ b)) ≡ cancel a b i) (deltaIntRet (a ⊖ b)) (deltaIntRet (suc a ⊖ suc b))
  test = λ i j → compPath-filler (deltaIntRet (a ⊖ b)) (cancel a b) i j
```
then I get a termination error.

File: Cubical/HITs/Ints/DeltaInt/Properties.agda [hunk 1]
```diff
@@ -41,30 +41,23 @@ cancelDiamond a b i j = hcomp (λ k → λ
...
+deltaIntRet (cancel 0 (suc b) i) = cancelTriangle 0 (suc b) i
 deltaIntRet (cancel (suc a) (suc b) i) = deltaIntRet (cancel a b i) ∙ cancelDiamond a b i
 
 DeltaInt≡Int : DeltaInt ≡ Int
 DeltaInt≡Int = isoToPath (iso toInt fromInt deltaIntSec deltaIntRet)
--}
 
 succPred : ∀ n → succ (pred n) ≡ n
 succPred (x ⊖ y) i = cancel x y (~ i)
--- cancel (suc a) (suc b) i ≡ cancel a b i
```
Comment (ice1000) [inline Cubical/HITs/Ints/DeltaInt/Properties.agda]: Why is this line removed?

Comment (ecavallo) [reply Cubical/HITs/Ints/DeltaInt/Properties.agda]: oh, I cleaned it out accidentally

Comment (ice1000) [reply Cubical/HITs/Ints/DeltaInt/Properties.agda]: Oh! This is a comment! I didn't notice this. I thought it's one clause of the function. I apologize.

**Comment by ice1000** [comment]
Review ping for @mortberg

**Pull request merged**